readme.md

```diff
@@ -0,0 +1,43 @@
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+## Portfolio: mobile version skeleton"
+
+>>This project was fitted into mobile with responsive dimension, you can find on this clear custom design an approach on your next portfolio work.
+
+
+
+## Built With
+
+- HTML/CSS
+- Visual Studio Code
+- Flexbox
+- Figma
+- responsive dimension 320x570
```

Comment:
- [ ] You should add major languages, frameworks, and technologies in this section as in the README [template]( https://github.com/microverseinc/readme-template/blob/master/README.md) provided by Microverse. However, these items does not belong to any of them.